mirage/README

```diff
@@ -0,0 +1,26 @@
+Helpful hints for mirage
+
+# mirage/config.ts file Routes
+  ## must be grouped withing the correct namespace and urlPrefix
+  ## osfResource should be singlular
+  ## osfNestedResource 
+    ### the parent resource name should match the osfResource name
+    ### the relationshipName should follow the singular/plural form on the app/model
+  ## the :parentID should be capitalized
+
+# any component router.ts file
+  ## resource name in a `findAll` should match the singluar/plural form of the osfResource
+  ## relationshipName name in a `queryHasMany` should match the singluar/plural form of the osfNestedResource
+
+# mirage/serializers file
+  ## buildNormalLinks `self` should match the url request
+  ## buildRelationships
+    ### attributes should match the singular/plural form on the app/model
+
+# mirage/fixtures file
+  ## plural file names
+  ## MirageModelRegistry
+    ### the attribute should be singular and hyphenated?
+  ## MirageSchemaRegistry
+    ### the attribute should be singular and hyphenated?
+  ## all fixtures need to be loaded in the mirage/scenarioes/default.ts file
```

Comment:
Can I make a small addition here? I've been bit by this a couple of times in the past when loading a fixture in the default scenario and then again in a specific scenario, mirage will throw an error that says something akin to `Error: <some_id> is already being used.` and it's not the most obvious error to trace down.

```suggestion
  ## all fixtures need to be loaded in the mirage/scenarioes/default.ts file
  ## fixtures should only be loaded once, so use caution when loading fixtures in a specific/non-default scenario
```